Amazon.QLDB.Driver.Tests/AsyncQldbDriverTests.cs

```diff
@@ -54,11 +55,11 @@ public void SetupTest()
                 .WithAWSCredentials(new Mock<AWSCredentials>().Object)
                 .WithQLDBSessionConfig(new AmazonQLDBSessionConfig());
             Assert.IsNotNull(builder);
-            
+
```

Comment:
While I applaud the cleaning of the code, in the future can we separate the formatting changes from the substantive code changes? It makes it easier to review the actual changes to the code.

Reply:
This is done automatically by settings in my IDE, and when they're not done IntelliSesnse gives you warnings. I think the last person who worked on this did not set this up even though in their environment.